Core/src/IfcGeometryConverter/GeometrySettings.h

```diff
@@ -163,8 +164,75 @@ namespace OpenInfraPlatform
 						dAngle += M_TWOPI;
 					}
 				}
+				/*!Compare two doubles for equality to the custom precision
+				*param[in]    first      The first value
+				*param[in]    second     The second value
+				*param[in]    precision  Precision of model
+				
+				*return true, if the absolute difference is smaller than the precision. False otherwise
+				*/
+				bool areEqual(const double first,const double second,const double precision) const
+				{
+					return abs(first - second) < precision;
+				}
+
+				/*!Compare two doubles for equality to the default precision
+				*param[in]    first      The first value
+				*param[in]    second     The second value
+
+				*return true, if the absolute difference is smaller than the precision. False otherwise
+				*/

```

Comment:
```suggestion
				/*!Compare two doubles for equality to the default precision
				* \param[in]    first      The first value
				* \param[in]    second     The second value
				*
				* \return true, if the absolute difference is smaller than the precision. False otherwise
				*/
```

Reply:
Resolved: 0be1668